libraries/exporters/src/main/java/org/eclipse/sw360/exporter/ReleaseExporter.java

```diff
@@ -109,6 +109,7 @@ private static void addToHeaders(List<String> headers, Release._Fields field) {
             case COMPONENT_ID:
                 headers.add(displayNameFor(field.getFieldName(), nameToDisplayName));
                 headers.add(displayNameFor("component type", nameToDisplayName));
+                headers.add(displayNameFor("Categories", nameToDisplayName));
```

Comment:
All other header names are starting with a small letter (at least if they are no acronym)

Reply:
Thanks for pointing it out.I have made the necessary changes.